main.go

```diff
@@ -130,6 +130,15 @@ func determineExportMethod(desiredExportMethod string, archiveExportMethod expor
 	return exportMethod, nil
 }
 
+func exportDSYMs(dsymDir string, dsyms []string) error {
+	for _, dsym := range dsyms {
+		if err := command.CopyDir(dsym, dsymDir, false); err != nil {
+			return fmt.Errorf("Failed to copy (%s) -> (%s), error: %s", dsym, dsymDir, err)
```

Comment:
Error strings should be lowercase and not contain "." as per golang standards. Additionally, "something, error: reason" is not a recommended pattern, use "sonething: reason" instead, as seen in the Wrap function (https://godoc.org/github.com/pkg/errors#Wrap)
```suggestion
			return fmt.Errorf("failed to copy (%s) -> (%s): %s", dsym, dsymDir, err)
```

Reply:
Many thanks for the clarification and the detailed answer!